crates/blockifier/src/execution/execution_utils.rs

```diff
@@ -68,35 +71,57 @@ pub fn execute_entry_point_call(
                 context,
             )
         }
-        ContractClass::V1(contract_class) => {
-            entry_point_execution::execute_entry_point_call(
-                call,
-                contract_class,
-                state,
-                resources,
-                context,
-            )
-        }
+        ContractClass::V1(contract_class) => entry_point_execution::execute_entry_point_call(
+            call,
+            contract_class,
+            state,
+            resources,
+            context,
+        ),
         ContractClass::V1Sierra(contract_class) => {
             let sierra_program: &SierraProgram = get_program(&contract_class);
             let contract_entrypoints: &ContractEntryPoints = get_entrypoints(&contract_class);
```

Comment:
What do you think of wrapping this logic inside a `native_entry_point_execution`? We match project style and get our own module where we put native logic